### PR TITLE
Bottom navigation added

### LINKED ISSUE #12: BottomNavigationView
- [x] Создать BottomNavigationView
- [x] Добавить 3 подпункта BottomNavigationView

### PR DESCRIPTION
**Pull request by kramarov-evg**
Closes #12 I've created bottom navigation for this app including 3 items
- Cryptocurrencies
- Currencies
- Stocks
Also changed starting activity in manifest, so now it'll be needed to migrate from activities to fragments

### CODE REVIEW

**Review by owlerxan — APPROVED** [review]
Noice

**Review by LeonidVolohov — APPROVED** [review]
Good job

**Review by owlerxan — COMMENTED** [review]
Названия иконок

**Comment by kramarov-evg** [comment]
> Названия иконок

Fixed

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
+
+    <com.google.android.material.bottomnavigation.BottomNavigationView
+        android:id="@+id/bottom_navigation"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        app:menu="@menu/bottom_navigation"
+        android:background="@color/design_default_color_background"/>
 
 </RelativeLayout>
```
Comment (owlerxan) [inline app/src/main/res/layout/activity_main.xml]: Codestyle

File: app/src/main/java/com/stonks/ui/DefaultFragment.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+class DefaultFragment : Fragment() {
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+//        return super.onCreateView(inflater, container, savedInstanceState)
+        return inflater.inflate(R.layout.default_fragment, container, false)
+    }
+}
```
Comment (owlerxan) [inline app/src/main/java/com/stonks/ui/DefaultFragment.kt]: Codestyle

File: app/src/main/res/layout/default_fragment.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        android:layout_height="wrap_content"
+        android:text="@string/fragment_placeholder_text"
+        android:textSize="@dimen/text_size_large"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:textAlignment="center"/>
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (owlerxan) [inline app/src/main/res/layout/default_fragment.xml]: Codestyle

File: app/src/main/res/menu/bottom_navigation.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        android:id="@+id/currency_tab"
+        android:icon="@drawable/ic_currency"
+        android:title="@string/currencies_tab_name"/>
+
+    <item
+        android:id="@+id/stocks_tab"
+        android:icon="@drawable/ic_stocks"
+        android:title="@string/stocks_tab_name"/>
+
+</menu>
```
Comment (owlerxan) [inline app/src/main/res/menu/bottom_navigation.xml]: Codestyle

**Review by owlerxan — APPROVED** [review]

**Comment by kramarov-evg** [comment]
@owlerxan, Fixed code style

@LeonidVolohov, fixed localizations

**Pull request merged**